### PR TITLE
Windows compatibility & utility improvements

### PR DESCRIPTION
**Pull request by wensi-ai**
- Add OG compatibility on windows
- Add installation support for zsh (linux) and cmd (windows)
- remove legacy config in pyproject.toml

### CODE REVIEW

**Comment by cgokmen** [comment]
I have added Windows compatibility and tempfile cleanup fixes to this PR as well.

**Review by ChengshuLi — APPROVED** [review]
LGTM. @cremebrule Would you mind taking another quick look?

File: omnigibson/utils/asset_utils.py
Comment (cremebrule) [inline omnigibson/utils/asset_utils.py:21]: nit, maybe we should rename this function so it semantically matches this docstring? e.g.: `dot_folder`

File: omnigibson/__init__.py
Comment (cremebrule) [inline omnigibson/__init__.py:109]: why is this needed?

**Review by cremebrule — COMMENTED** [review]
overall LGTM! A few comments, once addressed, feel free to merge!

File: omnigibson/__init__.py
Comment (wensi-ai) [reply omnigibson/__init__.py:109]: This is isaac sim bug on windows where this extension is not automatically loaded and will crash OG

File: omnigibson/utils/asset_utils.py
Comment (wensi-ai) [reply omnigibson/utils/asset_utils.py:21]: done

File: omnigibson/__init__.py
Comment (wensi-ai) [reply omnigibson/__init__.py]: On Linux the decrypted file will immediately get deleted after load so there shouldn't be an issue there.
On windows however, Isaac Sim won't let go of those usd files so we can only delete it upon shutdown, so I guess it should be put here so that og knows what the folder path is?

Comment (cremebrule) [reply omnigibson/__init__.py]: hmmm I see. @ChengshuLi what do you think? will turbosquid folks get mad at us if we do this? lol

**Pull request merged**
